locale/fr.json

```diff
@@ -0,0 +1,2122 @@
+{
+    "login": {
+        "copy": {
+            "translation": "Si vous avez déjà un compte human-connection, connectez-vous ici.",
+            "notes": ""
+        },
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
```

Comment:
> [<img alt="Elweyn" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Elweyn) **Authored by [Elweyn](https://github.com/Elweyn)**
_<time datetime="2019-11-18T12:24:19Z" title="Monday, November 18th 2019, 1:24:19 pm +01:00">Nov 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,2122 @@
+{
+    "login": {
+        "copy": {
+            "translation": "Si vous avez déjà un compte human-connection, connectez-vous ici.",
+            "notes": ""
+        },
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
```


"Votre courriel" seems a bit strange, I think it should be "Votre mail" or "Votre addresse mail"